src/node/build/buildPluginCss.ts

```diff
@@ -128,16 +127,10 @@ export const createBuildCssPlugin = ({
         return {
           code: modules
             ? dataToEsm(modules, { namedExports: true })
-            : (cssCodeSplit
-                ? // If code-splitting CSS, inject a fake marker to avoid the module
-                  // from being tree-shaken. This preserves the .css file as a
-                  // module in the chunk's metadata so that we can retrieve them in
-                  // renderChunk.
-                  `${cssInjectionMarker}()\n`
-                : ``) + `export default ${JSON.stringify(css)}`,
+            : `export default ${JSON.stringify(css)}`,
```

Comment:
Should this be 

```suggestion
            : `export default ${JSON.stringify(minifyCSS(css))}`,
```

So the app will get the same CSS that will eventually be written to the CSS file?

Reply:
https://github.com/vitejs/vite/issues/1249

Reply:
Thanks ! But it should be discuss inside other pr, not here.